calico-cloud/about/about-calico-cloud.mdx

```diff
@@ -0,0 +1,37 @@
+---
+description: A high-level description of Calico Cloud.
+---
+
+# About Calico Cloud
+
+## What is {{prodname}}?
+
+{{prodname}} is security solution for cloud-native applications running on containers and Kubernetes. It is the SaaS/pay-as-you-go version of Calico Enterprise, which also includes the core Calico Open Source (Calico CNI networking and network policy).
+
+![calico-cloud](/img/calico/calico-cloud.svg)
+
+Beyond the **Kubernetes security** features that you get from Calico Enterprise and Calico Open Source, {{prodname}} adds these **container security** solutions:
+
+- **Image vulnerability management**
+    Automated image scanning and blocking so you can monitor and assess workloads for new and existing CVEs 24/7. 
+
+- **Runtime container threat defense**
+    Fully-automated protection against known and unknown attacks (network or container-based.    
+
+## Best fit
+
+The sweetspot for {{prodname}} is small teams who need to manage the full spectrum of compliance in a web-based console. To jumpstart learning for teams, {{prodname}} provides:
+
+- Built-in onboarding tutorials in the web console
+- Automatic policy recommendations to make it easy for developers to secure their microservices and applications from day one
+- Hands-on training from Customer Support during the trial period so you can see how to realize your use cases in short order
+- Self-service workshops and training to speed up adoption
+
+## Need more info?
+
+- For specific {{prodname}} features, see [Tigera product comparison](/product-comparison.mdx)
+- To connect your cluster to {{prodname}} in 15 minutes, [start a free trial]()
+- [Calico Cloud pricing](https://www.tigera.io/tigera-products/calico-cloud-pricing/)
+- [Connect a cluster to {{prodname}} documentation](/get-started/connect/connect-cluster.mdx)
```

Comment:
Relative file links. All of these should be something like `(../get-started/bla-bla.mdx)`